webpack/scenes/Subscriptions/Details/SubscriptionDetails.js

```diff
@@ -63,32 +67,64 @@ class SubscriptionDetails extends Component {
             resource,
           }}
         />}
-        <div>
-          <LoadingState loading={subscriptionDetails.loading} loadingText={__('Loading')}>
-            <Row>
-              <Col sm={6}>
-                <SubscriptionDetailInfo
-                  subscriptionDetails={subscriptionDetails}
-                />
-              </Col>
-              <Col sm={6}>
-                <SubscriptionDetailAssociations
-                  subscriptionDetails={subscriptionDetails}
-                />
-                <SubscriptionDetailProducts
-                  subscriptionDetails={subscriptionDetails}
-                />
-              </Col>
-            </Row>
-          </LoadingState>
-        </div>
-      </Grid>
+
+        <TabContainer id='subscription-tabs-container' defaultActiveKey={1}>
+          <div>
+            <LoadingState loading={subscriptionDetails.loading} loadingText={__('Loading')}>
+              <Nav bsClass="nav nav-tabs">
+                <NavItem eventKey={1}>
+                  <div>{__('Details')}</div>
+                </NavItem>
+                <NavItem eventKey={2}>
+                  <div>{__('Enabled Products')}</div>
+                </NavItem>
+              </Nav>
+              <Grid bsClass="container-fluid">
+                <TabContent animation={false}>
```

Comment:
using react router on tabs content may increase cohesion and reduce the snapshot size.

Reply:
I'm wondering about this too since the tabs don't display new pages